src/sxastarter/scripts/temp/.npmignore

```diff
@@ -0,0 +1,2 @@
+*
```

Comment:
Can you double-check this file? I see that in _nextjs_ base template we have _gitignore_. As far as I remember npm doesn't include _gitignore_ into the package

Reply:
Indeed, good catch - this is a bug in JSS 21.1.0. As agreed, we have logged it to our backlog, and this will be resolved with the next release of JSS (21.1.1).